### ISSUE TITLE
Ion overwrites `astro.config.mjs`

### ISSUE THREAD

**Issue by garysassano** [sst/ion#440]
1. You have an existing Astro project
2. You execute `sst init`
3. `astro.config.mjs` gets transformed into a broken state

`astro.config.mjs` (before `sst init`)
```astro
import { defineConfig } from "astro/config";

import preact from "@astrojs/preact";

// https://astro.build/config
export default defineConfig({
  site: "https://example.com",
  integrations: [preact()]
});
```

`astro.config.mjs` (after `sst init`)
```astro
import { defineConfig } from "astro/config";
import aws from "astro-sst";

export default defineConfig({
  output: "server",
  adapter: aws(),
});
```
![image](https://github.com/sst/ion/assets/10464497/3e34bb93-a14f-4504-bbba-ad7156f6951d)

**Comment by jayair** [comment]
Yeah the `sst init` command is mostly designed for new projects. But it would be good if we could handle this.

**Comment by thdxr** [comment]
we have a few issues open like this - should we instead print what the user needs to add to the config instead of adding it ourselves?

**Comment by jayair** [comment]
Yeah probably. Maybe just do it automatically for Next.js for now?
